riscv/processor.cc

```diff
@@ -536,6 +539,11 @@ void state_t::reset(processor_t* const proc, reg_t max_isa)
   if (proc->extension_enabled(EXT_ZCMT))
     csrmap[CSR_JVT] = jvt = std::make_shared<jvt_csr_t>(proc, CSR_JVT, 0);
 
+  if (proc->extension_enabled(EXT_ZICFISS)) {
+    reg_t ssp_mask = (xlen == 64 ? 0xfffffffffffffff8ULL : 0xfffffffffffffffCULL);
```

Comment:
It's aesthetic, but I think this would be easier for readers of the code to rewrite this as `reg_t ssp_mask = -reg_t(xlen == 64 ? 8 : 4);`

Reply:
Thanks, I think it's more readable.